### PR TITLE
Add powershell-ls entry

### PR DESCRIPTION
**Pull request by AmjadHD**

### CODE REVIEW

**Comment by AmjadHD** [comment]
I don't know why test are failing ?

**Comment by jwortmann** [comment]
Thanks, it works with this configurations! But perhaps replace
```"-BundledModulesPath", "D:/Amjad/PowerShellEditorServices"```
with
```"-BundledModulesPath", "PATH/TO/PowerShellEditorServices"```

File: docs/index.md [hunk 1]
```diff
@@ -313,12 +313,57 @@ Or instead of LSP-julia, add the following client configuration:
...
+    "PATH/TO/PowerShellEditorServices/PowerShellEditorServices/Start-EditorServices.ps1",
+    "-LogPath", "PATH/TO/pses.log",  // the path itself is not relevant
+    "-LogLevel", "Normal",
+    "-SessionDetailsPath", "PATH/TO/session.json",  // the path itself is not relevant
+    "-FeatureFlags", "@()",
+    "-HostName", "'Sublime Text'",
+    "-HostProfileId", "subl",
+    "-HostVersion", "1.0.0",
+    "-AdditionalModules", "@()",
+    "-BundledModulesPath", "D:/Amjad/PowerShellEditorServices",
```
Comment (tomv564) [inline docs/index.md]: As @jwortmann suggested, can you change to 'PATH/TO' as you did elsewhere?

Comment (AmjadHD) [reply docs/index.md]: Missed that one, it's fixed now.

**Comment by tomv564** [comment]
Thanks @AmjadHD !

**Pull request merged**